src/coreclr/vm/methodtable.h

```diff
@@ -3401,8 +3403,11 @@ public :
 
         enum_flag_IsTrackedReferenceWithFinalizer   = 0x04000000,
 
-        enum_flag_HasCriticalFinalizer        = 0x08000000, // finalizer must be run on Appdomain Unload
-        enum_flag_Collectible                 = 0x10000000,
+        // TODO, andrewau, move fields to reflect value changes
```

Comment:
What is this TODO for?

Reply:
I wonder if I should make `enum_flag_HasCriticalFinalizer` part of `WFLAGS_LOW_ENUM` and therefore need change for the code above.